feature-group-definitions/font-palette.yml

```diff
@@ -0,0 +1,11 @@
+name: font-palette and @font-palette-values
+spec:
+  - https://drafts.csswg.org/css-fonts-4/#font-palette-prop
+  - https://drafts.csswg.org/css-fonts-4/#font-palette-values
+usage_stats: https://chromestatus.com/metrics/css/timeline/popularity/709
+compat_features:
+  - css.at-rules.font-palette-values
+  - css.at-rules.font-palette-values.base-palette
+  - css.at-rules.font-palette-values.font-family
+  - css.at-rules.font-palette-values.override-colors
+  - css.properties.font-palette
```

Comment:
Perhaps this should have `CSSFontPaletteValuesRule` and descendants too?

Reply:
Done above.